src/api/services/form/form.v6.service.js

```diff
@@ -0,0 +1,9 @@
+angular.module('ovh-api-services').service('OvhApiServicesFormV6', ($resource) => $resource('/services/:serviceId/form/:formName', {
```

Comment:
No cache ?

Reply:
Copy/paste from parent I guess, not entirely sure it is needed as main usage is for this is to open link from email meaning that a new app is started

Reply:
It's your usage of it, not all the future usage of it 😱

Reply:
I guess if cache is needed at one point, it can always be added

Reply:
😅